safehttp/plugins/hsts/hsts_test.go

```diff
@@ -24,120 +24,122 @@ import (
 	"github.com/google/go-safeweb/safehttp/safehttptest"
 )
 
-func TestHSTS(t *testing.T) {
+func TestHSTSReject(t *testing.T) {
```

Comment:
I've split these to not test for the `Body` always because it's up to the `Dispatcher`, not really the `Interceptor`.